rdkit-stubs/Chem/rdchem.pyi

```diff
@@ -95,7 +95,7 @@ class Atom:
     @classmethod
     def GetQueryType(cls, RDKit) -> Any: ...
     @classmethod
-    def GetSmarts(cls, RDKit) -> Any: ...
+    def GetSmarts(cls, doKekule: Optional[bool]=False, allHsExplicit: Optional[bool]=False, isomericSmiles: Optional[bool]=True) -> str: ...
```

Comment:
```suggestion
    def GetSmarts(cls, doKekule: Optional[bool]=..., allHsExplicit: Optional[bool]=..., isomericSmiles: Optional[bool]=...) -> str: ...
```

Maybe we should avoid setting the default values in these type stubs since we are not actually the source of truth.

Reply:
I've been using `...` aka `Undefined` to avoid accidentally telling lies but idk.

Reply:
I did source these defaults from their documentation. I'm inclined towards including it so that it makes it easier to inspect in an IDE. Ultimately, we can't be too worried about drift, otherwise we'll be in for a world of hurt since this isn't automatically generated right now.